samza-core/src/main/scala/org/apache/samza/job/local/ProcessJobFactory.scala

```diff
@@ -19,28 +19,43 @@
 
 package org.apache.samza.job.local
 
-
-import java.io.File
-
 import org.apache.samza.SamzaException
-import org.apache.samza.config.{JobConfig, Config}
+import org.apache.samza.checkpoint.CheckpointManagerUtil
+import org.apache.samza.clustermanager.ClusterBasedJobCoordinator
+import org.apache.samza.config.{Config, JobConfig}
 import org.apache.samza.config.TaskConfig._
 import org.apache.samza.coordinator.JobModelManager
+import org.apache.samza.coordinator.stream.{CoordinatorStream, CoordinatorStreamSystemConsumer, CoordinatorStreamSystemProducer}
 import org.apache.samza.job.{CommandBuilder, ShellCommandBuilder, StreamJob, StreamJobFactory}
+import org.apache.samza.metrics.MetricsRegistryMap
+import org.apache.samza.storage.ChangelogPartitionManager
 import org.apache.samza.util.{Logging, Util}
 
 /**
  * Creates a stand alone ProcessJob with the specified config.
  */
 class ProcessJobFactory extends StreamJobFactory with Logging {
-  def  getJob(config: Config): StreamJob = {
+  def getJob(config: Config): StreamJob = {
     val containerCount = JobConfig.Config2Job(config).getContainerCount
 
     if (containerCount > 1) {
       throw new SamzaException("Container count larger than 1 is not supported for ProcessJobFactory")
     }
-    
-    val coordinator = JobModelManager(config)
+
+    val metricsRegistry = new MetricsRegistryMap()
+    val coordinatorStream = new CoordinatorStream(config, metricsRegistry, getClass.getSimpleName)
+    coordinatorStream.startConsumer()
```

Comment:
Where are these stopped?

Reply:
The subclasses of `AbstractCoordinationManager` do it, but I see where you're going at. See the comment above regarding the CoordinatorStream.